### DISCUSSION TITLE
Add Clip Skip to Parameters Data

### DISCUSSION THREAD

**Discussion by gobshyte** [vladmandic/sdnext#600] [Ideas]
Hi
Wondering if it be possible to add Clip Skip into the PNG parameters? 
You get different results with different Clip Skip, and there is no way to tell (as far as I know) from exisitng parameter data which was used?
Sorry if this has already been suggested.

**Comment by EfourC** [comment]
The Clip Skip value is already stored in the image metadata (I see it when viewing the .png in a text editor).

When I just tested reading the image settings back in from the `Process Image` tab, it copied the Clip Skip value into the `Override settings` area.

![LHwBwetmLa](https://user-images.githubusercontent.com/121468033/235248852-bd40c8f9-0446-4eed-a596-73789375f67d.png)

Are you updated to the latest commit? I believe there were just some changes made to convert Clip Skip to be a 'local' value instead of global.

**Comment by tkiikt** [comment]
PNG parameters include the following.
**Prompt
 Negative Prompt
 Steps
 Sampler
 CFG scale
 Seed
 Size
 Model hash
 Model
 VAE
 Clip skip
 ENSD**

If you drop an image in Process Image tab, you will see the parameters in lower right corner.

**Comment by gobshyte** [comment]
Aha! My bad, sorry I can see it now, I am guessing it only appears if Clip Skip is set above 1? 
I can see it when Clip Skip is set to 2, but not when it's set to 1

**Reply by vladmandic** [reply]
exactly

**Comment by EfourC** [comment]
Interestingly, I see "Clip skip: 1" in the metadata as well, which is new behavior (I know it didn't used to).

![notepad++_sT4yhdC02p](https://user-images.githubusercontent.com/121468033/235259617-5d797157-74ec-4d6d-9aeb-ad3a8c5c3861.png)

![lTZpk7sdY3](https://user-images.githubusercontent.com/121468033/235259872-49760d04-d784-4022-b1bf-7609856e3e28.png)
